internal/cmdrpkgcopy/command.go

```diff
@@ -111,22 +117,59 @@ func (r *runner) runE(cmd *cobra.Command, args []string) error {
 		ObjectMeta: metav1.ObjectMeta{
 			Namespace: *r.cfg.Namespace,
 		},
-		Spec: porchapi.PackageRevisionSpec{
-			PackageName:    r.target,
-			Revision:       r.revision,
-			RepositoryName: r.repository,
-			Tasks: []porchapi.Task{
-				{
-					Type: porchapi.TaskTypeEdit,
-					Edit: &r.copy,
-				},
-			},
-		},
+		Spec: *revisionSpec,
 	}
+	pr.Spec.Tasks = []porchapi.Task{{Type: porchapi.TaskTypeEdit, Edit: &r.copy}}
 	if err := r.client.Create(r.ctx, pr); err != nil {
 		return errors.E(op, err)
 	}
-
 	fmt.Fprintf(cmd.OutOrStdout(), "%s created", pr.Name)
 	return nil
 }
+
+func (r *runner) getPackageRevisionSpec() (*porchapi.PackageRevisionSpec, error) {
+	res := resource.NewBuilder(r.cfg).
```

Comment:
Do we need to fetch the data here in table format? It seems like just using client-go directly would be easier.

Reply:
changed to client-go! Would appreciate if someone could take a look at it carefully, as I'm not familiar with client-go.